scripts/download_technique_animations.py

```diff
@@ -98,7 +98,7 @@ def gif_to_frames(filepath: str) -> None:
         animation_name = process_animation_name(base_name)
         for frame in range(0, image.n_frames):
             frame_filename = os.path.join(
-                ANIMATION_DIR, f"{base_name}{frame:02}.png"
+                ANIMATION_DIR, f"{base_name}_{frame:02}.png"
```

Comment:
The code already allows full stop as a delimiter, but I'm fine with adding _

Reply:
I wasn't aware of the full stop. I think that would be okay, but on some platforms the whole part after a full stop could be treated as a file extension so perhaps `_` is safer?